themes/metal/metal.theme.bash

```diff
@@ -73,7 +80,10 @@ function winname {
 
 # Displays the current prompt
 function prompt_command() {
-  PS1="\n${icon_start}$(virtualenv_prompt)${icon_user}${bold_green}\u${normal}${icon_host}${bold_cyan}\h${normal}${icon_directory}${bold_purple}\W${normal}\$([[ -n \$(git branch 2> /dev/null) ]] && echo \" on ${icon_branch}  \")${white}$(scm_prompt_info)${normal}\n${icon_end}"
+  #PS1="\n${icon_start}$(virtualenv_prompt)${icon_user}${bold_green}\u${normal}${icon_host}${bold_cyan}\h${normal}${icon_directory}${bold_purple}\W${normal}\$([[ -n \$(git branch 2> /dev/null) ]] && echo \" on ${icon_branch}  \")${white}$(scm_prompt_info)${normal}\n${icon_end}"
+  #PS2="${icon_end}"
+  #PS1="\[${green}\]AWS [\[${red}\]$AWS_DEFAULT_PROFILE\[${green}\]]:\[${reset}\] \w $ "
+  PS1="\n${icon_start}$(virtualenv_prompt)${icon_user}${bold_green}\u${normal}${icon_host}${bold_cyan}\h\[${green}\] ${icon_cloud} [\[${bold_red}\]$AWS_DEFAULT_PROFILE\[${green}\]]:\[${reset}\]${icon_directory}${bold_purple}\W${normal}\$([[ -n \$(git branch 2> /dev/null) ]] && echo \" on ${icon_branch}  \")${white}$(scm_prompt_info)${normal}\n${icon_end}"
```

Comment:
What happens if the user does not have the `aws-profile` tool installed? Can you please make support for this function optional?

Reply:
if the user doesn't have the aws_profile it doesn't show anything extra :+1:

Reply:
That's not true. I tried your updated theme (I don't have the `aws-profile` tool installed), and the theme added the cupcake, an empty set of brackets and a colon - see screenshot.

![bash-it-metal-theme-cloud](https://user-images.githubusercontent.com/907063/54592119-fbcbd700-4a2b-11e9-8724-c10389750b79.jpg)

Also, something seems to be off with the color escaping. As you can see, the original prompt (before your changes) switches back to white color for the dash, while with your change, the dash is now green.

Can you please make this new functionality optional, so that it does not interfere with the current functionality?